app/scripts/filters/resources.js

```diff
@@ -1012,6 +1012,14 @@ angular.module('openshiftConsole')
       return 'PAUSED_PENDING_INPUT' === stage.status;
     };
   })
+  .filter('strategyNameToParams', function () {
+    return function(strategyName) {
+      if (!strategyName) {
+        return null;
+      }
+      return strategyName.toLowerCase() + 'Params';
```

Comment:
We shouldn't assume this will always work. I'd rather have a switch statement for all the strategies we know about.